css-text/Overview.bs

```diff
@@ -1324,6 +1327,11 @@ Canonical order: N/A
       Shaping characters are still shaped as if the word were not
       broken, and grapheme clusters must stay together as one unit.
       No hyphenation character is inserted at the break point.
+    <dt><dfn>break-spaces</dfn></dt>
+    <dd>Line breaks are handled identically to ''overflow-wrap/normal'',
+      except that any sequence of <a>preserved</a> white space
+      that would otherwise overflow the line under these rules
+      must be broken after the last white space character that would fit the line.
```

Comment:
So when you say
> sequence of preserved white space...must be broken

you want to break between preserved white spaces but not before/after? Can you try to clarify?